### PR TITLE
Public Damage

### PR DESCRIPTION
**Pull request by BionicFrog**
Added public damage float and bool to ProjectileProperties and a small change to BulletCE to make it return the float

### CODE REVIEW

**Comment by AluminumAlman** [comment]
Would it be possible to just add `public int damageAmountBase;` or `public float damageAmountBase;` to `ProjectilePropertiesCE` and have it work out instead?

File: Source/CombatExtended/CombatExtended/Projectiles/ProjectilePropertiesCE.cs [hunk 1]
```diff
@@ -11,6 +11,8 @@ namespace CombatExtended
     public class ProjectilePropertiesCE : ProjectileProperties
     {
         // public float armorPenetration = 0;
+        public bool ShouldUsePublicDamage = false;
+        public float PublicDamage = 1;
```
Comment (kbatbouta) [inline Source/CombatExtended/CombatExtended/Projectiles/ProjectilePropertiesCE.cs:15]: Both of this should start with lowercase char.

**Review by kbatbouta — COMMENTED** [review]
Why is this even done at all? what is the reasoning behind adding this new field?

**Comment by BionicFrog** [comment]
> Why is this even done at all? what is the reasoning behind adding this new field?

The idea was to make it easier for modders to change damage of their projectiles, but rn it’s done in a weird way so I’ll probably close the request and try to improve it

**Pull request closed without merging**